scripts/setup/cursor.sh

```
@@ -29,87 +21,98 @@ setup_cursor() {
+    log_installed "Cursor"
+
     # ターゲットディレクトリの作成
-    log_info "Cursor設定ディレクトリを作成します (存在しない場合): $target_user_dir"
-    mkdir -p "$target_user_dir"
+    mkdir -p "$cursor_target_dir"
 
     # 設定ファイルのシンボリックリンクを作成
+    local linked_count=0
     for file in "$config_dir"/*; do
         if [ -f "$file" ]; then
             local filename=$(basename "$file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)